wiki/osu!_wiki/Contribution_guide/Best_practices/en.md

```diff
@@ -0,0 +1,83 @@
+# Best practices
+
+This page covers tasks and chores that you may face while contributing. The approaches mentioned here are designed to make the process easier and may be applied to other projects hosted on GitHub or similar platforms.
+
+## Introduction
+
+**Git** is a version control system which helps manage changes to files. The osu! wiki's data and history of changes are stored in a Git repository. **GitHub** is a platform for development that provides a web interface for Git repositories and offers a set of tools for project management.
+
+## Syncing the fork
+
+In order to make changes to a repository located on GitHub, a potential contributor needs to obtain a controlled copy of it called a *fork*. When you create your fork of the `osu-wiki` repo, you take a snapshot of its contents at this very moment. To make a meaningful contribution, **always sync your fork** before making a set of changes—this can be done directly from GitHub:
```

Comment:
```suggestion
In order to make changes to a repository located on GitHub, a potential contributor needs to obtain a controlled copy of it called a *fork*. When you create your fork of the `osu-wiki` repo, you take a snapshot of its contents at this very moment. To make a meaningful contribution, you must **always sync your fork** before making a set of changes—fortunately, this can be done directly from GitHub:
```